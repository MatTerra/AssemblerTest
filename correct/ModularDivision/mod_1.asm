mod_1: begin
section data
result: space
section text
n: extern
mod_2: extern
public finish
public divide
input n
input n
jmp mod_2
divide: load n
div n
store result
output result
finish: stop
end
