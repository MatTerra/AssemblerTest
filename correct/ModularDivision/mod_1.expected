> 2
PC <- 2
ACC <- 0

> 4
PC <- 4
ACC <- 0

PC <- 16
ACC <- 0

PC <- 18
ACC <- 4

PC <- 20
ACC <- 4

PC <- 6
ACC <- 4

PC <- 8
ACC <- 4

PC <- 10
ACC <- 1

PC <- 12
ACC <- 1

PC <- 14
ACC <- 1
OUTPUT: 1

PC <- 15
ACC <- 1

